lib/msal-node/src/client/ConfidentialClientApplication.ts

```diff
@@ -218,15 +217,13 @@ export class ConfidentialClientApplication
         }
     }
 
-    private setClientCredential(configuration: Configuration): void {
```

Comment:
this is a bug. configuration's type never should have been set to `Configuration`. It should've been, and now is, `NodeConfiguration`.